pubspec.yaml

```diff
@@ -53,6 +54,10 @@ dev_dependencies:
   hive_generator: '0.7.0+2'
   build_runner: '1.8.1'
 
+dependency_overrides:
+  dartx: ^0.3.0
+  characters: ^0.5.0
+
```

Comment:
This needs to be discarded before merging the PR. 
I had issues to resolve all the dependencies (tried with latest Flutter stable, beta, dev, master).

Reply:
I think this is needed, hit the pub version resolve error myself today: https://github.com/hivedb/hive/issues/203